provision/ubuntu/install.sh

```diff
@@ -57,6 +57,7 @@ cd ..
 rm -fr util-linux-2.30.1/ util-linux-2.30.1.tar.gz
 
 # Documentation dependencies
+sudo apt-get install -y libenchant1c2a
```

Comment:
Could you add this one in the line 44 please? So we keep all in the same action.

Reply:
But this is a documentation dependency, aren't we splitting the dependencies by usage?

Reply:
No not really. If all can be in the same apt-get install I use to add in there.

Reply:
done